ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+
+workflow:
+  as: openshift-qe-node-density-cni
+  steps:
+    test:
+      - ref: openshift-qe-node-density-cni
+      - ref: openshift-qe-write-perfscale-results
```

Comment:
The run looks to be failing because you need to put the output to a file. You can use this environment variable [here](https://github.com/openshift/release/blob/22b629bfeb14ac22ddb4517b1d75fc00565a1415/ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-chain.yaml#L7) 
You'll need to set it in the envionrment variable set up of the workload: https://github.com/openshift/release/blob/22b629bfeb14ac22ddb4517b1d75fc00565a1415/ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-ref.yaml#L8
And add the following lines in your command script: https://github.com/openshift/release/blob/22b629bfeb14ac22ddb4517b1d75fc00565a1415/ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh#L30-L31